### PR TITLE
Use log consistently and improve formatting

### PR DESCRIPTION
**Pull request by cyberdelia**
This should improve debugging and also readability.

### CODE REVIEW

File: plans.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ func GetPlan(name string) Plan {
 		return Plan{200}
 	case "ika", "2":
 		return Plan{400}
-	case "zilla", "baku", "mecha", "ryu", "4"," 6", "7":
+	case "zilla", "baku", "mecha", "ryu", "4", " 6", "7":
```
Comment (will) [inline plans.go]: The space before the 6 is probably a mistake

File: checks.go [hunk 2]
```diff
@@ -127,18 +127,17 @@ func idleQueriesCheck(db *sqlx.DB) Check {
...
-	Index_scan_pct  string `json:"index_scan_pct"`
-	Scans_per_write string `json:"scans_per_write"`
-	Index_size      string `json:"index_size"`
-	Table_size      string `json:"table_size"`
+	Reason        string `json:"reason"`
+	Index         string `json:"index"`
+	IndexScanPCT  string `json:"index_scan_pct"`
+	ScansPerWrite string `json:"scans_per_write"`
+	IndexSize     string `json:"index_size"`
+	TableSize     string `json:"table_size"`
```
Comment (will) [inline checks.go]: I think it has to be _, or you have to go through and change all the sql so it knows how to fill in the struct with data

Comment (cyberdelia) [inline checks.go]: Yeah, I need to add  `db:"table_size"` to the struct too

**Comment by cyberdelia** [comment]
I added the db mapping.

**Comment by will** [comment]
Ok, assuming that db mapping is actually a thing, which I will trust you that it is, this seems fine.

**Comment by cyberdelia** [comment]
@will It's a feature of [sqlx](https://github.com/jmoiron/sqlx#usage)

**Comment by will** [comment]
I'll take your word for it

**Pull request merged**